### ISSUE TITLE
bug: new chat input context and send button keyboard focus & navigability

### ISSUE THREAD

**Issue by toolmantim** [sourcegraph/cody#1842]
### Version

main

### Describe the bug

The new chat input context is missing focus styles for keyboard accessibility, and the send button isn't keyboard focusable:

https://github.com/sourcegraph/cody/assets/153/19624953-f4e1-4d53-9a42-d1f506d8011c

### Expected behavior

They have same style as the built-in VS Code search toggle buttons:

https://github.com/sourcegraph/cody/assets/153/ee733854-17d8-48f4-b1c1-73e23d17639e

and you should be able to tab to the send button from the context button.
